Add known issues to saved-objects migrations (#99388)
elastic/kibana: docs/CHANGELOG.asciidoc

// end::notable-breaking-changes[]

[float]
[[known-issues-v7.12.0]]
=== Known issues

> When upgrading to 7.12.0, Kibana will retry failed saved object migrations without user intervention.

Reviewer: ```suggestion
When upgrading to 7.12.0, {kib} retries failed saved object migrations without user intervention.
```

> There are known issues, including users who tried the fleet beta where migrations might fail.

Reviewer: I'm a little out of my area here, but is this a repeat of line 218? If so, can we delete it?

Author: done

Impacts include:

> * There is a known issue in v7.12.0 for users who tried the fleet beta. Upgrade migrations fail because

Reviewer: ```suggestion
* In 7.12.0, upgrade migrations fail because of a large number of documents in the `.kibana` index when you use **Fleet**.
```

> of a large number of documents in the `.kibana` index.

Reviewer: ```suggestion
```

* Corrupt saved objects caused by manual editing or integrations.

> * User defined index templates that causes new `.kibana*` indices to have incompatible settings or mappings.

Reviewer: ```suggestion
* User-defined index templates that cause new `.kibana*` indices to have incompatible settings or mappings.
```

* An unhealthy {es} cluster.
* Different versions of {kib} connected to the same {es} index.
* Incompatible `xpack.tasks.index` configuration setting.

> If your upgrade migration fails repeatedly, please refer to {kibana-ref}/upgrade-migrations.html#preventing-migration-failures[preventing migration failures].

Reviewer: ```suggestion
If your upgrade migration fails repeatedly, refer to {kibana-ref}/upgrade-migrations.html#preventing-migration-failures[preventing migration failures].
```
